### ISSUE TITLE
RFC822 reference left in discussion of date formats (Trac #134)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#134]
There's one reference left to RFC 822 in the discussion of date formats (Section 3.3.1):

"The first format is preferred as an Internet standard and represents a fixed-length subset of that defined by [RFC1123] (an update to [RFC822])."

Can we just drop the reference to RFC 822 (which has been updated twice in the meantime)?

Migrated from https://trac.ietf.org/ticket/134
```json
{
    "status": "closed",
    "changetime": "2012-03-22T00:06:26",
    "_ts": "1332374786229633",
    "description": "There's one reference left to RFC 822 in the discussion of date formats (Section 3.3.1):\n\n\"The first format is preferred as an Internet standard and represents a fixed-length subset of that defined by [RFC1123] (an update to [RFC822]).\"\n\nCan we just drop the reference to RFC 822 (which has been updated twice in the meantime)?\n\n",
    "reporter": "julian.reschke@gmx.de",
    "cc": "",
    "resolution": "fixed",
    "time": "2008-11-09T21:08:26",
    "component": "p1-messaging",
    "summary": "RFC822 reference left in discussion of date formats",
    "priority": "normal",
    "keywords": "",
    "milestone": "06",
    "owner": "",
    "type": "editorial",
    "severity": "Active WG Document"
}
```

**Comment by mnot** [comment]
Trac update at 20081110T21:49:30: fielding@gbiv.com commented:

> I think we should drop that reference.
>

**Comment by mnot** [comment]
Trac update at 20081114T01:22:24: mnot@pobox.com changed milestone from "unassigned" to "06"

**Comment by mnot** [comment]
Trac update at 20081114T20:09:03: 
* julian.reschke@gmx.de commented:

> Fixed in [384]:
> 
> Resolve #134: remove unneeded reference to RFC 822 (closes #134)


* julian.reschke@gmx.de changed resolution from "" to "fixed"
* julian.reschke@gmx.de changed status from "new" to "closed"

**Comment by mnot** [comment]
Trac update at 20120322T00:06:26: 
* mnot@pobox.com changed priority from "" to "normal"
* mnot@pobox.com changed severity from "Candidate WG Document" to "Active WG Document"
